packages/markdown/processor/plugin/section-anchor-id.js

```diff
@@ -1,17 +0,0 @@
-const flatMap = require('unist-util-flatmap');
```

Comment:
Make sure you remove this dependency if it's not being used anymore! (i can't remember if i added it or not)